content/docs/started/workstation/appliances/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Workstation Appliances"
+description = "Detailed instructions for leveraging VM appliances for Kubeflow"
```

Comment:
I really like the structure based on the operating system. That's why I think that the `Workstation Appliances` section in not needed here. A user will just select the preferred guide based on their OS.

Reply:
I can move this directory to the reference section .. otherwise it'll remain in this subtree. Also, there might be a better word (ie not Appliance). Any thoughts on that?

Reply:
+1, I think this section should be moved out of the subtree for clarity sake

As for the naming, I've never heard the term appliance used in this context. What about something like "Workstation Kubernetes" or "Local Dev Environments"?

Reply:
Okay, I'll move this directory to Reference, and I'll change the name to Local Dev Environments. Let's see how that looks.

Reply:
Actually, I'm thinking Virtual Developer Environments seems most appropriate.